tests/Feature/Users/Registration/AccountCreationTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Tests\Feature;
+
+use DB;
+use Hash;
+use Mockery;
+use Tests\TestCase;
+
+class AccountCreationTests extends TestCase {
+    public function testRegister() {
+        $response = $this->post('/user/register/',  $this->userAttributes());
+
+        $response->assertStatus(302);
+        $response->assertRedirect('dashboard');
```

Comment:
Could be worth a $this->assertDatabaseHas('users', [
        'email' => 'sally@example.com'
    ]);

to provide some basic test that the user is being created?

Reply:
Added.  

As a habit I tend to avoid dipping into the internals to test thing when we're trying to test the interface, as it can lead to you doing that just because the interface is a bit awkward, and then ending up with weaker interface testing.